lib/support/init.d/gitlab.default.example

```diff
@@ -29,3 +29,9 @@ web_server_pid_path="$pid_path/unicorn.pid"
 # sidekiq_pid_path defines the path in which to create the pid file for sidekiq
 # The default is "$pid_path/sidekiq.pid"
 sidekiq_pid_path="$pid_path/sidekiq.pid"
+
+# shell_path defines the path of shell for "$app_user" in case you disabled
+# shell of "$app_user" by commands like `usermod -s /sbin/nologin $app_user"
```

Comment:
@drunkard I feel that this doesn't belong here the way it was written down now. Can you please change it to:

```
shell_path defines the path of shell for "$app_user" in case you are using 
shell other than "bash"
```